setup.py

```diff
@@ -29,22 +29,31 @@ def read(fname):
     return open(os.path.join(os.path.dirname(__file__), fname)).read()
 
 
+DEPENDENCIES = [
+    "Twisted>=19.2.1",
+    "prometheus_client>=0.7.0,<0.8",
+    "aioapns>=1.7",
+    "pyyaml>=5.1.1",
+    "service_identity>=18.1.0",
+    "zope.interface>=4.6.0",
+    "idna>=2.8",
+    "jaeger-client>=4.0.0",
+    "opentracing>=2.2.0",
+]
+
+EXTRAS = {"sentry": ["sentry-sdk>=0.10.2"], "firebase": ["firebase-admin>=3.0.0"]}
```

Comment:
Do we want to bother doing this? Doesn't it make it more annoying to install if you have to do `pip install sygnal[all]` to actually get support for firebase?